export/main.ts

```diff
@@ -26,6 +29,12 @@ async function fetchProjectData(projectId: string, group?: string): Promise<any>
             console.log(await response.text());
         } else {
             const data = await response.json();
+            const latestJobId = data.latestJobId;
+
+            if(latestJobId) {
+                return evaluatePolicy(latestJobId, policy);
```

Comment:
Does this need to be awaited? Or does it not matter because the calling code will await it? My JS is a bit rusty...